metricflow/test/fixtures/semantic_manifest_yamls/simple_manifest/metrics.yaml

```diff
@@ -578,3 +578,35 @@ metric:
       - name: bookings
         offset_window: 14 days
         alias: bookings_2_weeks_ago
+---
+metric:
+  name: "bookings_offset_once"
+  description: bookings metric offset once.
+  type: derived
+  type_params:
+    expr: 2 * bookings
+    metrics:
+      - name: bookings
+        offset_window: 5 days
+---
+metric:
+  name: "bookings_offset_twice"
+  description: bookings metric offset twice.
+  type: derived
+  type_params:
+    expr: 2 * bookings_offset_once
+    metrics:
+      - name: bookings_offset_once
+        offset_window: 5 days
```

Comment:
Can we make this offset value something different, maybe 2 days or 8 days? It's a little hard to pick up which offset is which with the INTERVAL values being the same.